src/main/java/com/atlassian/jira/cloud/jenkins/util/IssueKeyExtractor.java

```diff
@@ -33,7 +33,7 @@ private IssueKeyExtractor() {
             KEY_PREFIX_REGEX + KEY_BODY_REGEX + KEY_POSTFIX_REGEX;
     private static final Pattern PROJECT_KEY_PATTERN = Pattern.compile(ISSUE_KEY_REGEX);
 
-    private static final Integer ISSUE_KEY_MAX_LIMIT = 100;
+    public static final Integer ISSUE_KEY_MAX_LIMIT = 100;
```

Comment:
Do we now use this constant in the `ChangeLogExtractor`? They should have a common interface and this constant should be defined on the interface level